### ISSUE TITLE
simpler flask+react template

### ISSUE THREAD

**Issue by benswift** [ACT-COVID-19-TRACKER/hospital-demand-webapp#7]
This repo (forked from <https://github.com/briancappello/flask-react-spa>) is pretty "battteries included" (which I like) but some of those batteries we don't need (which makes me more nervous). I'm open to switching if we find something which better suits our needs.

Here are a list of pros & cons:

### Pros

- uses ansible (automated setup ftw)
- docker-compose seems to give a nice local dev experience
- lots of fancy js bundling/code splitting stuff (which would be important for a big frontend SPA)
- has sqlalchemy (postgres) support in case we need to CRUD things
- nice libs for building easy APIs (flask-marshmallow)
- user authentication, session management stuff (using Flask-Security)
- has a principled approach to dev/prod (controlled using environment variables, logging levels set appropriately etc)

### Cons

- lots of fancy js bundling/code splitting stuff (not so important for our app, adds complexity)
- not sure we need the celery/redis
- some of the deps are a big out of date (e.g. flask, webpack) and I'm not keen to change the versions since I know these ones have all been designed to work nicely together
- basically just lots of moving parts, some of which I don't understand very well
